src/flutter/shell/platform/linux_embedded/plugin/lifecycle_plugin.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Is this correct that this copyright is different from the one in lifecycle_plugin.cc?

Reply:
It's copypasta. I fixed now.

Reply:
thx